src/block/tile/EnderChest.php

```diff
@@ -27,6 +27,19 @@
 
 class EnderChest extends Spawnable{
 
+	protected int $viewerCount = 0;
+
+	public function getViewerCount() : int{
+		return $this->viewerCount;
+	}
+
+	public function setViewerCount(int $viewerCount) : self{
+		if($viewerCount < 0)
+			throw new \UnexpectedValueException($viewerCount . ' is too small of a value');
+		$this->viewerCount = $viewerCount;
+		return $this;
```

Comment:
Tiles don't usually have fluent APIs (particularly since plugins are supposed to avoid them in PM4).